group-coordinator/src/main/java/org/apache/kafka/coordinator/group/streams/StreamsGroup.java

```diff
@@ -251,7 +249,7 @@ public String stateAsString(long committedOffset) {
     public ListGroupsResponseData.ListedGroup asListedGroup(long committedOffset) {
         return new ListGroupsResponseData.ListedGroup()
             .setGroupId(groupId)
-            .setProtocolType(ConsumerProtocol.PROTOCOL_TYPE)
+            .setProtocolType("streams")
```

Comment:
This is fine, but do we want to consider an `enum` for streams related requests?

Reply:
I think for classic groups, there is more than one protocol (consumer, sr, connect...) but for streams groups, there is only a single protocol, that's why I added a literal here. So for a single value, not sure if an enum makes sense. But we could move it to a static constant.